src/bionics.cpp

```diff
@@ -272,7 +288,7 @@ void player::power_bionics()
     const int tabs_start = 1;
     const int tab_step = 3;
 
-    while(true) {
+    for (;;) {
```

Comment:
I too wish it didn't complain for naked literals... preprocessor macros, ok; variables, ok, but a literal "true" is certainly sigh worthy, I agree. *For* what it's worth, `for (;;)` *for* that reason seems to be about 50/50 vs. `while (true)` in more recent code bases. I'm personally fine with either -- I just wish the compiler were too.